src/inmanta/module.py

```diff
@@ -1518,40 +1518,41 @@ class ProjectMetadata(Metadata, MetadataFieldRequires):
     :param copyright: (Optional) Copyright holder name and date.
     :param modulepath: (Optional) This value is a list of paths where Inmanta should search for modules.
     :param downloadpath: (Optional) This value determines the path where Inmanta should download modules from
-      repositories. This path is not automatically included in the modulepath!
+        repositories. This path is not automatically included in the modulepath!
     :param install_mode: (Optional) This key determines what version of a module should be selected when a module
-      is downloaded. For more information see :class:`InstallMode`.
+        is downloaded. For more information see :class:`InstallMode`.
     :param repo: (Optional) A list (a yaml list) of repositories where Inmanta can find modules. Inmanta tries each repository
-      in the order they appear in the list. Each element of this list requires a ``type`` and a ``url`` field. The type field
-      can have the following values:
+        in the order they appear in the list. Each element of this list requires a ``type`` and a ``url`` field. The type field
+        can have the following values:
 
-      * git: When the type is set to git, the url field should contain a template of the Git repo URL. Inmanta creates the
-        git repo url by formatting {} or {0} with the name of the module. If no formatter is present it appends the name
-        of the module to the URL.
-      * package: When the type is set to package, the URL field should contains the URL of the Python package repository.
-        The repository should be `PEP 503 <https://www.python.org/dev/peps/pep-0503/>`_ (the simple repository API) compliant.
+        * git: When the type is set to git, the url field should contain a template of the Git repo URL. Inmanta creates the
+            git repo url by formatting {} or {0} with the name of the module. If no formatter is present it appends the name
+            of the module to the URL.
 
-      The old syntax, which only defines a Git URL per list entry is maintained for backward compatibility.
+        * package: When the type is set to package, the URL field should contain the URL of the Python package repository.
+            The repository should be `PEP 503 <https://www.python.org/dev/peps/pep-0503/>`_ (the simple repository API)
+            compliant.
+
```

Comment:
This doesn't render well in the documentation

![Screenshot from 2023-01-17 13-23-21](https://user-images.githubusercontent.com/2684622/212900391-e7b7bb24-ece7-4397-a073-394938bc19f5.png)

Reply:
![image](https://user-images.githubusercontent.com/98876549/213185566-477915d7-a7db-4914-a42a-5519aef8213f.png)